### PR TITLE
Second batch of new code examples

### PR DESCRIPTION
**Pull request by AnashOommen**

### CODE REVIEW

File: examples/ErrorHandling/HandleRateExceededError.cs [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+                {
+                    // Checks if any of the errors are QuotaError.RESOURCE_EXHAUSTED or
+                    // QuotaError.RESOURCE_TEMPORARILY_EXHAUSTED.
+                    if (e.Failure.Errors.Where(err =>
+                        err.ErrorCode.QuotaError == QuotaError.ResourceExhausted ||
+                        err.ErrorCode.QuotaError == QuotaError.ResourceTemporarilyExhausted)
+                        .Count() > 0)
+                    {
+                        Console.Error.WriteLine(
+                            $"Received rate exceeded error, retry after {retrySeconds} seconds.");
```
Comment (CoryLiseno) [inline examples/ErrorHandling/HandleRateExceededError.cs]: Should we also demonstrate how to use the Scope of the error like we did in https://developers.google.com/adwords/api/docs/samples/csharp/error-handling#handle-rate-exceeded-errors?

Comment (CoryLiseno) [inline examples/ErrorHandling/HandleRateExceededError.cs]: Should we also demonstrate how to use the Scope of the error like we did in https://developers.google.com/adwords/api/docs/samples/csharp/error-handling#handle-rate-exceeded-errors?

Comment (AnashOommen) [reply examples/ErrorHandling/HandleRateExceededError.cs]: The scope is not available in API yet.

File: examples/ErrorHandling/HandleRateExceededError.cs [hunk 2]
```diff
@@ -0,0 +1,192 @@
+// Copyright 2019 Google LLC
```
[the remark this answers is not in the document]
Comment (AnashOommen) [reply examples/ErrorHandling/HandleRateExceededError.cs]: Done.

File: examples/Extensions/UpdateFeedItemAttributeValue.cs [hunk 3]
```diff
@@ -0,0 +1,314 @@
+﻿// Copyright 2019 Google LLC
```
[the remark this answers is not in the document]
Comment (AnashOommen) [reply examples/Extensions/UpdateFeedItemAttributeValue.cs]: Done

File: examples/Extensions/UpdateFeedItemAttributeValue.cs [hunk 4]
```diff
@@ -0,0 +1,314 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using static Google.Ads.GoogleAds.V2.Enums.FlightPlaceholderFieldEnum.Types;
+
+namespace Google.Ads.GoogleAds.Examples.V2
+{
+    /// <summary>
+    /// This code example updates a FeedItemAttributeValue in a flights feed. To create a flights
+    /// feed, run the AddFlightsFeed.cs. This code example is specific to feeds of type
+    /// DYNAMIC_FLIGHT.The attribute you are updating must be present on the feed. This code
```
[the remark this answers is not in the document]
Comment (AnashOommen) [reply examples/Extensions/UpdateFeedItemAttributeValue.cs]: Done.

File: examples/Extensions/UpdateFeedItemAttributeValue.cs [hunk 5]
```diff
@@ -0,0 +1,314 @@
...
+            // Gets the feed resource name.
+            string feedResourceName = ResourceNames.Feed(customerId, feedId);
+
+            // Gets a map of the placeholder values and feed attributes.
+            Dictionary<FlightPlaceholderField, FeedAttribute> feedAttributes =
+                GetFeed(client, customerId, feedResourceName);
+
+            // Gets the ID of the attribute to update. This is needed to specify which
+            // FeedItemAttributeValue will be updated in the given FeedItem.
+
```
[the remark this answers is not in the document]
Comment (AnashOommen) [reply examples/Extensions/UpdateFeedItemAttributeValue.cs]: Done.

File: examples/ErrorHandling/HandleRateExceededError.cs [hunk 6]
```diff
@@ -0,0 +1,192 @@
...
+using static Google.Ads.GoogleAds.V2.Enums.AdGroupCriterionStatusEnum.Types;
+using static Google.Ads.GoogleAds.V2.Enums.KeywordMatchTypeEnum.Types;
+using static Google.Ads.GoogleAds.V2.Errors.QuotaErrorEnum.Types;
+
+namespace Google.Ads.GoogleAds.Examples.V2
+{
+    /// <summary>
+    /// This code example demonstrates how to handle RateExceededError in an application.
+    /// This code example runs 5 threads in parallel, each thread attempting to validate
+    /// 100 keywords in a single request.While spanning 5 parallel threads is unlikely to
```
[the remark this answers is not in the document]
Comment (AnashOommen) [reply examples/ErrorHandling/HandleRateExceededError.cs]: Done.

File: examples/ErrorHandling/HandleRateExceededError.cs [hunk 7]
```diff
@@ -0,0 +1,192 @@
...
+        public static void Main(string[] args)
+        {
+            HandleRateExceededError codeExample = new HandleRateExceededError();
+            Console.WriteLine(codeExample.Description);
+            try
+            {
+                // The Google Ads customer ID for which the call is made.
+                long customerId = long.Parse("INSERT_CUSTOMER_ID_HERE");
+
+                // the ad group ID to which keywords are added.
```
[the remark this answers is not in the document]
Comment (AnashOommen) [reply examples/ErrorHandling/HandleRateExceededError.cs]: Done.

File: examples/ErrorHandling/HandleRateExceededError.cs [hunk 8]
```diff
@@ -0,0 +1,192 @@
...
+        /// <summary>
+        /// Returns a description about the code example.
+        /// </summary>
+        public override string Description
+        {
+            get
+            {
+                return "This code example demonstrates how to handle RateExceededError in an " +
+                    "application. This code example runs 5 threads in parallel, each thread " +
+                    "attempting to validate 100 keywords in a single request.While spanning 5" +
```
[the remark this answers is not in the document]
Comment (AnashOommen) [reply examples/ErrorHandling/HandleRateExceededError.cs]: Done.

**Pull request merged**